test/mock/adapter/adapter.go

```diff
@@ -246,7 +246,7 @@ func listenForDIDCommMsg(actionCh chan service.DIDCommAction, store storage.Stor
 				ID:   uuid.NewString(),
 				Name: "Demo Verifier",
 				InputDescriptors: []*presexch.InputDescriptor{
-					{ID: uuid.NewString(), Schema: []*presexch.Schema{{URI: "https://w3id.org/citizenship#PermanentResidentCard"}}},
+					{ID: uuid.NewString(), Schema: []*presexch.Schema{{URI: "https://www.w3.org/2018/credentials#VerifiableCredential"}}},
```

Comment:
Can we change issuer to save PR card than switching this in Verifier ? PR card is the primary flow used in Sandbox and it would be good if we can have same flow set here as well.

Reply:
@rolsonquadras : created https://github.com/trustbloc/wallet/issues/1504 for this, I will update make this change this sprint.